controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -533,33 +533,17 @@ private boolean streamCutEqualOrAfter(Map<Long, Long> cut1, Map<StreamSegmentRec
     private List<ScaleMetadata> mapToScaleMetadata(List<EpochRecord> epochRecords) {
         final AtomicReference<List<StreamSegmentRecord>> previous = new AtomicReference<>();
         return epochRecords.stream()
-                           .map(record -> {
-                               long splits = 0;
-                               long merges = 0;
-                               List<StreamSegmentRecord> segments = record.getSegments();
-                               if (previous.get() != null) {
-                                   splits = findSegmentSplitsMerges(previous.get(), segments);
-                                   merges = findSegmentSplitsMerges(segments, previous.get());
-                               }
-                               previous.set(segments);
-                               return new ScaleMetadata(record.getCreationTime(), transform(segments), splits, merges);
-                           }).collect(Collectors.toList());
-    }
-
-    /**
-     * Method to calculate number of splits and merges.
-     *
-     * Principle to calculate the number of splits and merges:
-     * 1- An event has occurred if a reference range is present (overlaps) in at least two consecutive target ranges.
-     * 2- If the direction of the check in 1 is forward, then it is a split, otherwise it is a merge.
-     *
-     * @param referenceSegmentsList Reference segment list.
-     * @param targetSegmentsList Target segment list.
-     * @return Number of splits/merges.
-     */
-    private long findSegmentSplitsMerges(List<StreamSegmentRecord> referenceSegmentsList, List<StreamSegmentRecord> targetSegmentsList) {
-        return referenceSegmentsList.stream().filter(
-                segment -> targetSegmentsList.stream().filter(target -> target.overlaps(segment)).count() > 1 ).count();
+                .map(record -> {
```

Comment:
i dont see any changes here other than indentation, please revert this.

Reply:
fixed